services/pkg/man-cgi/run.nginx

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec nginx
```

Comment:
using a custom run file here because the nginx container has `daemon off` in the config and the packaged nginx runit service has `nginx -g daemon off;`, so nginx complains about the duplication